<launch>
  <node name="motion_node" type="pandora_vision_motion_node" pkg="pandora_vision_motion"
    respawn="False" output = "screen"/>
  
  <rosparam file="$(find pandora_vision_motion)/config/motion_topics.yaml"
    command="load" ns="motion"/>
  
  <rosparam file="$(find pandora_vision_motion)/config/motion_params.yaml"
    command="load" ns="motion"/>
    
</launch>
